python/bin/thunder-streaming

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ -z "$THUNDER_STREAMING_PATH" ]; then
+    echo "You need to set the THUNDER_STREAMING_PATH environment variable" 
+    exit 1
+fi
+ipython -i $THUNDER_STREAMING_PATH/python/front-end/shell/shell.py
```

Comment:
Probably replace with:

export PYTHONSTARTUP = $THUNDER_STREAMING_PATH/python/front-end/shell/shell.py
python

So we're not dependent on ipython